xbout/load.py

```diff
@@ -181,6 +181,12 @@ def _trim(ds, ghosts={}, keep_guards=True):
 
     trimmed_ds = trimmed_ds.drop(_BOUT_TIMING_VARIABLES, errors='ignore')
 
+    # Ignore FieldPerps for now
+    for name in trimmed_ds:
+        if (trimmed_ds[name].dims == ('x', 'z')
```

Comment:
Do FieldPerps not have any identifying features other than their literal dimensions? Like an attributes flag?

Reply:
`FieldPerp`s do have an attribute `yindex_global` which other fields don't have, but I think at the moment the convention for identifying `Field3D`, `Field2D`, `FieldPerp` in netCDF files is by their dimensions. In the current setup this is unambiguous; are you thinking it'll cause problems when adding 'physical' coordinates, etc.? It would be easy to add an attribute like `bout_type` - this is already done in BOUT++ for the HDF5 input/output because HDF5 variables don't have dimensions that we could use to identify them.

Reply:
Okay that seems fine for now then